### PR TITLE
Document using container names for management

### PR DESCRIPTION
**Pull request by mwhipple**
The current GroovyDoc for `DockerExistingContainer` does not mention the Docker provided support for using a container name instead of ID and an example allowing for managing  of a Docker container across Gradle invocations has been added.

Provides solution for case outlined in #562

### CODE REVIEW

**Comment by orzeh** [comment]
LGTM :+1: Thanks!

File: README.asciidoc [hunk 1]
```diff
@@ -447,6 +447,50 @@ task functionalTestMyApp(type: Test, dependsOn: startMyAppContainer) {
...
+
+task dockerCreateContainer(type: DockerCreateContainer) {
+    dependsOn buildMyAppImage
+    targetImageId { buildMyAppImage.imageId }
+    containerName = CONTAINER_NAME
+}
+
+task dockerStartContainer(type: DockerStartContainer) {
+    dependsOn dockerCreateContainer
+    targetContainerId { CONTAINER_NAME }
```
Comment (cdancy) [inline README.asciidoc]: Lets change this to: `targetContainerId { dockerCreateContainer.getContainerId() }`.

Lets do this for all the tasks below as well so folks know they can chain things together using the outputs of one task as the inputs to another in true gradle fashion.

**Comment by cdancy** [comment]
One minor comment to address but overall LGTM.

**Comment by mwhipple** [comment]
Updated to use task property.

**Comment by cdancy** [comment]
Just as an FYI I've halted on merging this PR as I think our README is getting to be too big and unruly IMO. I've been looking to clean things up and streamline the documentation off to the side.

**Comment by bmuschko** [comment]
@mwhipple We introduce a more readable and maintainable user guide: https://github.com/bmuschko/gradle-docker-plugin/pull/630. Would you mind translating your changing to the user guide?

**Pull request closed without merging**